src/main/java/com/internet/shop/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -30,34 +31,32 @@ public Order create(Order order) {
             if (generatedKeys.next()) {
                 order.setOrderId(generatedKeys.getLong(1));
             }
+            List<Long> productIds = new LinkedList<>();
```

Comment:
do we use this list?
also, if we don't need the ResultSet anymore, let's close the statement before we pass the connection to a different method